templates/etc/td-agent/td-agent.conf

```diff
@@ -15,8 +15,8 @@
 <match **>
   type google_cloud
   # Set the chunk limit conservatively to avoid exceeding the limit
-  # of 10MB per write request.
-  buffer_chunk_limit 2M
+  # of 5MB per write request.
```

Comment:
The configured request size limit on the API side is 10MB. But the recommended size is 5MB. I've fixed the comment, PTAL.

I agree that letting the plugin count the encoded bytes would be a better solution here, but that's a feature request for another day (let's open a bug to track).